### PR TITLE
Add test for OpenLibrary search method

### PR DESCRIPTION
**Pull request by marcegl91**
### Summary

- Add tests for OpenLibrary search method with WebMock

### Trello Card

https://trello.com/c/S3MDg5ty/38-crear-tests-para-el-servicio-de-openlibrary

### CODE REVIEW

File: wbooks-api/spec/support/mocks_and_stubs/open_library_request_invalid.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module OpenLibraryRequestInvalid
+  def self.call
```
Comment (facucompe) [inline wbooks-api/spec/support/mocks_and_stubs/open_library_request_invalid.rb]: I would send the isbn code as a parameter here

File: wbooks-api/spec/support/mocks_and_stubs/open_library_request_success.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module OpenLibraryRequestSuccess
+  def self.call
```
Comment (facucompe) [inline wbooks-api/spec/support/mocks_and_stubs/open_library_request_success.rb]: same here

File: wbooks-api/spec/support/mocks_and_stubs/open_library_request_success.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module OpenLibraryRequestSuccess
```
Comment (facucompe) [inline wbooks-api/spec/support/mocks_and_stubs/open_library_request_success.rb]: I don't know if I would create another module for success response. You don't have to repeat yourself in the URL method for example.

**Review by facucompe — CHANGES_REQUESTED** [review]

File: wbooks-api/spec/services/open_library_service_spec.rb [hunk 4]
```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+require 'support/mocks_and_stubs/open_library_request'
```
Comment (facucompe) [inline wbooks-api/spec/services/open_library_service_spec.rb]: I think that if you add `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
` this won't be needed in every file

**Review by facucompe — APPROVED** [review]

**Review by floresv — APPROVED** [review]
Good job!

**Pull request merged**